apps/admin-gui/src/app/facilities/pages/resource-detail-page/resource-groups/resource-groups.component.html

```diff
@@ -1,10 +1,10 @@
 <h1 class="page-subtitle">{{'RESOURCE_DETAIL.ASSIGNED_GROUPS.TITLE' | translate}}</h1>
 <perun-web-apps-refresh-button (refresh)="loadAllGroups()"></perun-web-apps-refresh-button>
-<button mat-flat-button color="accent" (click)="addGroup()">
+<button (click)="addGroup()" *ngIf="!loadingResource && guiAuthResolver.isAuthorized('assignGroupsToResource_List<Group>_Resource_policy',[resource])" color="accent" mat-flat-button>
   {{'RESOURCE_DETAIL.ASSIGNED_GROUPS.ADD_GROUP' | translate}}
 </button>
-<button mat-flat-button color="warn" class="ml-2" [disabled]="selected.selected.length === 0" (click)="removeGroups()">
-  {{'RESOURCE_DETAIL.ASSIGNED_GROUPS.REMOVE_GROUP' | translate}}
+<button (click)="removeGroups()" [disabled]="selected.selected.length === 0 || !canRemoveGroups()" class="ml-2 mr-2" color="warn" mat-flat-button>
+  <span  [matTooltipDisabled]="canRemoveGroups()" matTooltip="{{'RESOURCE_DETAIL.ASSIGNED_GROUPS.REMOVE_TOOLTIP' | translate}}">{{'RESOURCE_DETAIL.ASSIGNED_GROUPS.REMOVE_GROUP' | translate}}</span>
```

Comment:
at least split this code into the - one "thing" one row, this is very hard to read